src/Tasks/ContentReviewEmails.php

```diff
@@ -57,6 +58,15 @@ protected function getOverduePagesForOwners(SS_List $pages)
                 continue;
             }
 
+            // get most recent review log of current [age]
+            $contentReviewLog = $page->ReviewLogs()->sort('Created DESC')->first();
+
+            // check log date vs NextReviewDate. If someone has left a content review
+            // after the review date, then we don't need to notify anybody
+            if ($contentReviewLog && $contentReviewLog->Created >= $page->NextReviewDate) {
+                continue;
```

Comment:
I think this change is good, and should merge nicely with #97 - however I think it would also be a great idea if a tidy up of `$record->advanceReviewDate()` was included here before `continue`ing :)